compiler/testSuite/Vifelse07.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/io;
+
```

Comment:
Isn't this already handled by Vifelse06?

Reply:
directly using the boolean value as if condition than using a stored value takes a different code path so this was added for code coverage